coralproject/talk
# [CORL-2473]: update member and moderator role change copy

## What does this PR do?

Updates member and moderator role change copy around assigning/removing sites. Also updates commenter bio copy in configuration.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Go to `General` --> `Commenter bios` and see it says `commenter` instead of `member`. Go to Role dropdown in the `Community` tab as a site moderator and see the updated text around assigning/removing sites from members/moderators.
 
 
## How do we deploy this PR?
